pilot/platform/kube/inject/configmap.go

```diff
@@ -1,54 +0,0 @@
-// Copyright 2017 Istio Authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package inject
-
-// NOTE: This tool only exists because kubernetes does not support
-// dynamic/out-of-tree admission controller for transparent proxy
-// injection. This file should be removed as soon as a proper kubernetes
-// admission controller is written for istio.
-
-import (
-	"fmt"
-
-	v1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/client-go/kubernetes"
-
-	meshconfig "istio.io/api/mesh/v1alpha1"
-	"istio.io/istio/pilot/model"
-)
-
-// GetMeshConfig fetches the ProxyMesh configuration from Kubernetes ConfigMap.
-func GetMeshConfig(kube kubernetes.Interface, namespace,
```

Comment:
oic, here it was but gone now

Reply:
Right. This was (1) an exported function that encouraged other components to fetch ConfigMap directly and (2) not required by the core injection code. The only real consumer was the `kube-inject` command so I moved there and refactored to consolidate the k8s client setup code.